package.json

```diff
@@ -5,9 +5,10 @@
   "main": "index.js",
   "scripts": {
     "build-transform": "npx token-transformer tokens/src/tokens.json tokens/dist/tokens.json",
-    "build-sd": "style-dictionary build --config configs/all.json",
+    "build-sd": "style-dictionary build --config configs/config.json",
+    "build-sd-config": "node configs/build-config.js",
     "build-tw": "postcss ./styles.css -o ./tailwind.css",
-    "build": "npm run build-transform && npm run build-sd",
+    "build": "npm run build-sd-config && npm run build-transform && npm run build-sd",
```

Comment:
Are we using this for anything at this point @spreston4? If not we should go ahead and remove them in favor of the action.yml since that is where our script is most up to date

Reply:
No it's not being used by our action at all. I just set it up to help with development. Will remove.